### PR TITLE
Testing part1: Pair-programming

### PR DESCRIPTION
**Pull request by Paulinelali**
With this PR we:

- [x] Added tests to addTask
- [x] Added test to removeTask
- [x] Updated package.json
- [x] Updated style.css
- [x] Include webpack
- [x] Added coverage feature to tests
- [x] reformated test files
- [x] Updated removeTask.test.js
- [x] Tried a lot to fix Webhint error but it is not being fixed. Please excuse.
- [x] Updated package.json to remove linter errors but it is not being removed.

### CODE REVIEW

File: tests/removeTask.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  it('should not add a new task if the input is empty', () => {
+    const tasks = [];
+    const taskInput = { value: '' };
+
+    const result = addTask(taskInput, tasks);
+
+    // Assert: Check that the array is still empty
+    expect(result).toEqual([]);
+  });
+});
```
Comment (amejid) [inline tests/removeTask.test.js]: - [ ] You did a great job testing the add item functionality. :+1: This entire file seems to be a duplicate of `addTask.test.js` and does not hold any tests regarding the delete functionality. Kindly ensure your tests also cover the delete functionality. 🧐️

**Review by amejid — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct workflow :heavy_check_mark: 
- Add item functionality covered in the test :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: tests/removeTask.test.js (hunk 1 above)
Comment (Mike111222) [reply tests/removeTask.test.js]: Thanks a lot sir for your kind feedback. I have overlooked it. I am really happy that you gave me such nice suggestions to improve my code. I have updated it as you mentioned. Thanks

Comment (peterrobert) [reply tests/removeTask.test.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Paulinelali, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**